packages/common/src/intl/locales/en/distribution.json

```
@@ -48,6 +51,9 @@
   "label.supply-quantity": "Quantity to supply",
   "label.units-in-pack-size-of_one": "unit in pack size of",
   "label.units-in-pack-size-of_other": "units in pack size of",
+  "messages.edit-service-charges": "Edit service charges",
+  "messages.stock-charges-description": "Total charges for lines related to stock. The tax amount is an effective tax rate calculated by the total tax paid over the subtotal of all stock lines.",
+  "messages.service-charges-description": "Total amounts of service charges. The tax amount is an effective tax rate calculated by the total tax paid over the subtotal of all service charges.",
```

```suggestion
  "messages.stock-charges-description": "Total charges for stock lines. The calculated tax amount is an effective tax rate using the total tax paid over the subtotal of all stock lines.",
  "messages.service-charges-description": "Total amounts of service charges. The calculated tax amount is an effective tax rate using the total tax paid over the subtotal of all service charges.",
```

tbh, just delighted to see the info tooltip. the words can be adjusted later..
Probably would be good to explain about tax per line, but since there isn't a way to edit that I think it would add to the confusion to have it in now! I was just trying to explain that the calculated amount is shown, until you edit the value.. because after than it isn't calculated, it is using what you put in 🤷